R/CDISCFilteredData.R

```diff
@@ -10,45 +10,31 @@
 #' key and only those rows that appear in the parent dataset are kept in the filtered
 #' dataset.
 #'
-#' The relational model is attached to the data. When `set_data` is called with the
-#' data, its `keys` attribute has the following elements:
-#'  - `primary`: columns based on which unique rows are counted
-#'  - `parent`: parent dataset based on which the dataset is filtered (via `inner_join`)
-#'  - `foreign`: keys to join on when filtering with parent dataset
-#'
 #' The teal UI works with objects of class `FilteredData` which may mix CDISC and other
 #' data (e.g. `iris`).
 #'
-#' It is not too difficult to make this class generalize to more than one parent dataset,
-#' provided each dataset then contains information about its parents together with the
-#' shared keys with each of them.
-#'
 #' @seealso `FilteredData` class
 #'
 #' @examples
 #' library(scda)
 #' library(shiny)
+#' library(teal.data)
```

Comment:
`shiny` will be loaded together with teal.data
https://github.com/insightsengineering/teal.data/blob/ee99ab4126b2a3523a14085cf1ed7d5451b93ce6/DESCRIPTION#L18-L20
```suggestion
#' library(teal.data)
```